Content/package.minimal.json

```diff
@@ -2,7 +2,7 @@
     "private": true,
     "scripts": {
         "start": "webpack-dev-server",
-        "build": "webpack -p"
+        "build": "cross-env NODE_ENV=production webpack -p"
```

Comment:
If this is "best practice" I'm wondering whether we should force it for minimal template - maybe just use the best practice for default template and let minimal users handle that themselves if necessary?

Reply:
I don't mind, I was just being consistent in applying it to both versions of the template as they both issue the same command but I can see your point. I'm happy to take it out of the minimal template? Let me know.

Reply:
yes please - let's keep the minimal minimal :)

Reply:
Hi, that's done. Thanks